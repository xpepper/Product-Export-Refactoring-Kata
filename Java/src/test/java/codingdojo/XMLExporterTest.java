package codingdojo;

import org.approvaltests.Approvals;
import org.junit.Test;

import static codingdojo.SampleModelObjects.RecentOrder;
import static java.util.Collections.emptyList;
import static java.util.Collections.singletonList;

public class XMLExporterTest {

    @Test
    public void export_no_orders() {
        String result = XMLExporter.exportFull(emptyList());
        Approvals.verify(result);
    }

    @Test
    public void export_with_one_order() {
        String result = XMLExporter.exportFull(singletonList(RecentOrder));
        Approvals.verifyXml(result);
    }
}
