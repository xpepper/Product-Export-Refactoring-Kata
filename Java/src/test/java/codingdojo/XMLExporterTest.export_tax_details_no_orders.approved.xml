<?xml version="1.0" encoding="UTF-8"?><orderTax>0.00</orderTax>
